assignments/array-methods.js

```diff
@@ -56,21 +56,47 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(function(runner){
+    fullName.push(`${runner.first_name} ${runner.last_name}`)
+});
+0
 console.log(fullName);
 
+
+
+
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+
+runners.map(function(runner){
+    allCaps.push(runner.first_name.toUpperCase());
```

Comment:
While this code functions just fine, you are treating the map() method the same as the forEach() method. The map method should return a brand new array. The callback function should return the changed value that enters into the array. What you need to do in this instance is to return `(runner.first_name.toUpperCase())` instead of pushing it into the allCaps array. You will also need to set the allCaps array equal to runners.map().